readers-frontend/src/components/Article.astro

```diff
@@ -50,31 +54,54 @@ const {
             <>
               <a class="h-full w-1/2" href={articleUrl}>
                 <img
-                  class="aspect-video rounded-xl object-cover"
+                  class="aspect-video w-full rounded-xl object-cover"
                   src={imageUrl}
                   alt="Article Thumbnail"
+                  onerror={`if (this.src !== ${imagePlaceHolder}) this.src = ${imagePlaceHolder}`}
                 />
               </a>
-              <div class="border-color ml-3 h-full w-1/2 border-t-2">
-                <a
-                  class="text-l category-text category-color uppercase xl:text-xl 2xl:text-2xl"
-                  href={`/${categoryLink}`}
-                >
-                  {category}
-                </a>
+              <div
+                class={`border-color ml-3 h-full w-1/2 ${
+                  !isInMuseBanner && "border-t-2"
```

Comment:
Instead of doing this, we should use something like `showCategoryAccent` because when we start adding the article pages we also won't show the category at the top yk